.github/workflows/pull-request.yml

```diff
@@ -79,6 +81,7 @@ jobs:
         run: make setup-infra
         env:
           PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+          STACK_NAME: dev
```

Comment:
Why we need this?

Reply:
Its not now, leftover from when i was doing a conditional on the domain.

Reply:
I removed this.